libobs/media-io/video-io.h

```diff
@@ -24,6 +24,8 @@
 extern "C" {
 #endif
 
+#define MAX_VIDEO_ENCODERS 4
```

Comment:
Just a reasonable sounding number

Reply:
I think MAX_VIDEO_STREAMS or MAX_VIDEO_TRACKS might be a better name. Number seems fine to me